global.json

```diff
@@ -2,7 +2,6 @@
   "$schema": "https://json.schemastore.org/global.json",
   "sdk": {
     "version": "8.0.204",
-    "rollForward": "latestMajor",
-    "allowPrerelease": true
+    "rollForward": "disable"
```

Comment:
Version rollover has been disabled following https://github.com/dotnet/runtime/pull/82905, which breaks locked-mode restore for consumers of ILLink between .NET SDK `8.0.204` and `8.0.205`.